kube-config-files/neo4j-red-service.yaml

```diff
@@ -0,0 +1,72 @@
+###############################################################################
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: neo4j-red
+  labels:
+    app: neo4j-red
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: neo4j-red
+  template:
+    metadata:
+      labels:
+        app: neo4j-red
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: neo4j-red
+        image: coco/ft-neo4j:v2.3.7-4
+        imagePullPolicy: Always
+        env:
+          - name: NEO4J_AUTH
+            value: none
+          - name: NEO4J_HEAP_MEMORY
+            value: "3584"
+          - name: NEO4J_CACHE_MEMORY
+            value: "1024m"
+        ports:
+        - containerPort: 7474
+        volumeMounts:
+         - name: neo4j-red-persistent-storage
+           mountPath: /data
+      volumes:
+       - name: neo4j-red-persistent-storage
+         awsElasticBlockStore:
+           volumeID: vol-08d4132f23b45de31
+           fsType: ext4
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: neo4j-red
+  labels:
+    app: neo4j-red
+    visualize: "true"
+spec:
+  ports:
+    - name: work
+      port: 7474
+  clusterIP: None
```

Comment:
I don't think this should be a headless service, as we want to use it to access the neo4j pod